martypy/ClientMV2.py

```diff
@@ -268,11 +319,22 @@ def sidestep(self, side: str, steps: int = 1, step_length: int = 35,
         return self.ricIF.cmdRICRESTRslt(f"traj/sidestep/{steps}?side={ClientGeneric.SIDE_CODES[side]}"
                                          f"&stepLength={step_length}&moveTime={move_time}")
 
+    def set_volume(self, volume: int) -> bool:
+        return self.ricIF.cmdRICRESTRslt(f"audio/vol/{volume}")
+
+    def get_volume(self) -> int:
+        result = self.ricIF.cmdRICRESTURLSync("audio/vol")
+        if result.get("rslt", "") == "ok":
+            return result.get("volPC", 0)
+        return 0
+
     def play_sound(self, name_or_freq_start: Union[str,float],
             freq_end: Optional[float] = None,
             duration: Optional[int] = None) -> bool:
+        if name_or_freq_start.lower().endswith(".mp3"):
+            return self.ricIF.play_mp3(name_or_freq_start)
         if not name_or_freq_start.lower().endswith(".raw"):
-                name_or_freq_start += ".raw"
+            name_or_freq_start += ".raw"
```

Comment:
Consider what will happen if we move to use mp3 files in firmware instead for sounds, to allow greater sample rates